src/sentry/static/sentry/app/components/events/interfaces/spans/dividerHandlerManager.tsx

```diff
@@ -145,8 +158,16 @@ export class Provider extends React.Component<PropType, StateType> {
 
     const dividerHandlePositionString = toPercent(this.dividerHandlePosition);
 
-    selectRefs(this.dividerLineRefs, (dividerDOM: HTMLDivElement) => {
-      dividerDOM.style.left = dividerHandlePositionString;
+    selectRefs(this.ghostDividerLineRefs, (dividerDOM: HTMLDivElement) => {
+      const {parentNode} = dividerDOM;
+
+      if (!parentNode) {
```

Comment:
Good point. TypeScript told me it can be null. So I'm covering for this case. I expect it shouldn't be null.

But this can be true `document.createElement('div').parentNode === null`.